### ISSUE TITLE
It would be nice to have `<Plug>(fern-action-git-status-toggle)` mapping

### ISSUE THREAD

**Issue by bluz71** [lambdalisue/fern-git-status.vim#5]
Hello @lambdalisue,

The `g:fern_git_status_disable_startup` flag is available to disable *fern-git-status* by default. That is useful, since git status, even with performance optimisation, can still be an **expensive** task.

I would like to have a mapping to allow the toggling of *fern-git-status* just as I currently have for toggling marks and hidden files:

```viml
nmap <buffer> m <Plug>(fern-action-mark-toggle)j
nmap <buffer> h <Plug>(fern-action-hidden-toggle)
```

I use these often.

If possible, I would like something similar for fern-git-status:

```viml
nmap <buffer> g <Plug>(fern-action-git-status-toggle)
```

When I hit `g` I get git-status details (due to `g:fern_git_status_disable_startup` being set to 1), even if this causes pause or slowdown. When I no longer want git-status details, maybe because performance is a little slower, I hit `g` again and they disappear as if the `fern-git-status` was never loaded to begin with. 

If this is too hard, just close this. But if possible, it would be very handy.

Best regards.

**Comment by lambdalisue** [comment]
Probably possible. I'll make it once the performance issue has solved.

**Comment by bluz71** [comment]
> I'll make it once the performance issue has solved.

Yes, good plan.

No rush here.

**Comment by bluz71** [comment]
After using the newest *fern-git-status* for a few days now I don't feel I need this anymore.

I will permanently use *fern-git-status* since performance is nearly always good with these settings:

```viml
let g:fern_git_status#disable_ignored = 1
let g:fern_git_status#disable_untracked = 1
let g:fern_git_status#disable_submodules = 1
``` 

I did an extreme test with the Linux Kernel repository which is 60,000+ committed files on my slowest machine, a 2015 12" Macbook with fanless Core-M processor (very slow processor).

Performance of *fern-git-status* was initially bad.
However, when I did the following in the Linux repository performance was then excellent:

```sh
git config feature.manyFiles true
```

That feature was introduced in [Git 2.24](https://github.blog/2019-11-03-highlights-from-git-2-24) and on slow machines with **HUGE** repositories it makes a very big difference, day and night.

For now I am closing this.
